nix/mysql/mysql_test.nix

```diff
@@ -7,6 +7,19 @@
       CREATE USER bar;
     '';
   };
+  services.mysql.m3 = {
+    enable = true;
+    importTimeZones = true;
+    package = pkgs.mysql80;
+    initialDatabases = [{ name = "test_dabase_with_tzdata"; }];
```

Comment:
I don’t see this being useful for the test below, is it needed?

Reply:
no, actually not needed. removing it.